Core/Graphics/CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
 set(SOURCES
     "Include/Babylon/Graphics.h"
+    "Source/PlatformSpecificHeaders/${BABYLON_NATIVE_PLATFORM}/GraphicsPlatform.h"
```

Comment:
@Drigax is already doing [something similar](https://github.com/BabylonJS/BabylonNative/pull/582/files?file-filters%5B%5D=.txt#diff-1c56f1023b5f001b105f47bb306a1daf4de97640a11e11d339fe5db298e9df17) in his PR. I much like the pattern in his PR than this pattern. Thoughts?

Reply:
Spoke offline, restructured folders per recommendation.